site/templates/default.html

```diff
@@ -8,8 +8,8 @@
       <meta content="haskell,functional,pure,programming,lazy" name="keywords">
       <meta content="The Haskell purely functional programming language home page." name="description">
       <link href="/img/favicon.ico" rel="shortcut icon">
-      <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro|Raleway:700,900|Ubuntu+Mono:400" type="text/css" rel="stylesheet">
-      <link href="/css/hl.css" type="text/css" rel="stylesheet">
+      <link rel="stylesheet" href="/css/fonts.css">
```

Comment:
`hl.css` is unruly at 4.7k+ lines of unformatted CSS, so I think for now separate stylesheets for the fonts is okay. In the future, maybe we can break out `hl.css` using something like ![Sass modules](https://sass-lang.com/guide/#modules) that can be compiled into a single CSS file at build time?

Reply:
Yes that would be nice to have an assets pipeline that deals with this.